server/lib/database.js

```diff
@@ -24,18 +24,47 @@ function readSQLFiles () {
 }
 
 module.exports.createTables = function createTables () {
+
 	const connection = getConnection();
 
-	return connection.then(function (connection) {
+	function createDB (conn){
 
-		const query = denodeify(connection.query.bind(connection));
+		const query = denodeify(conn.query.bind(conn));
```

Comment:
Completely errors out when recreating the tables because we're trying to handle values when none are returned.